diff --git a/serial/serialposix.py b/serial/serialposix.py
index 0bf3df5..c5af45a 100644
--- a/serial/serialposix.py
+++ b/serial/serialposix.py
@@ -270,7 +270,7 @@ class Serial(SerialBase, PlatformSpecific):
                 pass
             else:
                 raise
-        self.reset_input_buffer()
+#        self.reset_input_buffer()
         self.pipe_abort_read_r, self.pipe_abort_read_w = os.pipe()
         self.pipe_abort_write_r, self.pipe_abort_write_w = os.pipe()
         fcntl.fcntl(self.pipe_abort_read_r, fcntl.F_SETFL, os.O_NONBLOCK)
